src/collection-preferences/__tests__/compact-mode-table.test.tsx

```diff
@@ -0,0 +1,81 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I'm not sure where this would go, but I would also like a test where we render something more elaborate in a compact table. For example, inline editing.

Reply:
Good point, I'll add more variations of tables here.